includes/general.php

```diff
@@ -1024,6 +1041,23 @@ function pods_shortcode( $tags, $content = null ) {
  * @since 2.3.0
  */
 function pods_shortcode_form( $tags, $content = null ) {
+	try {
+		return pods_shortcode_form_run( $tags, $content );
+	} catch ( Exception $exception ) {
+		return $exception->getMessage();
+	}
+}
+
+/**
+ * Form Shortcode support for use anywhere that support WP Shortcodes
+ *
+ * @param array  $tags    An associative array of shortcode properties
+ * @param string $content Not currently used
+ *
+ * @return string
+ * @since 2.7.13
+ */
+function pods_shortcode_form_run( $tags, $content = null ) {
```

Comment:
@sc0ttkclark I assume this is a private function? Maybe prepend it with an underscore?

Reply:
We'll refactor this in the future when we expand shortcodes/blocks more. Not anything to worry about right now though.

Reply:
Future: Classes with private methods